### PR TITLE
chore: updated kubernetes dashboard

### LINKED ISSUE #9265: Update Kubernetes dashboard to match UX design
### Is your enhancement related to a problem? Please describe

In [this ticket](https://github.com/containers/podman-desktop/issues/9264) we will create the basic page for the Kubernetes welcome page. This task is to update the page to match the design that will be provider in [the design issue ](https://github.com/containers/podman-desktop/issues/9263#issue-2573273119).

### Describe the solution you'd like

[See design](https://github.com/containers/podman-desktop/issues/9263#issue-2573273119)

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by gastoner**
### What does this PR do?
Updates Kubernetes dashboard to match UX design

Following design from https://github.com/containers/podman-desktop/issues/9263

### Screenshot / video of UI
image

The Graphs needs to be discussed and added in another PR/Issue

### What issues does this PR fix or reference?
Closes #9265 

### How to test this PR?
- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Comment by cdrage** [comment]
Nitpick, for 
image


The top and bottom black lines are distracting, maybe we photoshop it out?

**Comment by gastoner** [comment]
@cdrage updated!

File: packages/renderer/src/lib/kube/KubernetesDashboard.svelte [hunk 1]
```diff
@@ -22,69 +24,138 @@ import IngressRouteIcon from '../images/IngressRouteIcon.svelte';
...
+                    <KubernetesDashboardResourceCard type='Deployments' Icon={DeploymentIcon} activeCount={activeDeploymentsCount} count={deploymentCount} link='/kubernetes/deployments'/>
+                    <KubernetesDashboardResourceCard type='Services' Icon={ServiceIcon} count={serviceCount} link='/kubernetes/services'/>
+                    <KubernetesDashboardResourceCard type='Ingresses & Routes' Icon={IngressRouteIcon} count={ingressRouteCount} link='/kubernetes/ingressesRoutes'/>
+                    <KubernetesDashboardResourceCard type='Persistent Volume Claims' Icon={PvcIcon} count={pvcCount} link='/kubernetes/persistentvolumeclaims'/>
+                    <KubernetesDashboardResourceCard type='ConfigMaps & Secrets' Icon={ConfigMapSecretIcon} count={configMapSecretCount} link='/kubernetes/configmapsSecrets'/>
+                </div>
+                <!-- Graphs -->
+                
+              {/if}
+              <!-- Articles and blog posts - colapsible -->
```
Comment (benoitf) [inline packages/renderer/src/lib/kube/KubernetesDashboard.svelte]:
```suggestion
              <!-- Articles and blog posts - collapsible -->
```

**Review by benoitf — APPROVED** [review]
note related to this PR / non-blocking, but hoping to have https://github.com/podman-desktop/podman-desktop/issues/9704 soon because we should not have the data on the frontend side.

**Comment by gastoner** [comment]
@cdrage @deboer-tim PTAL 🙏

**Comment by deboer-tim** [comment]
Tested, found a couple issues:

1. If you scroll up, the header and tiles have almost the same color so they blend together:
Screenshot 2024-11-27 at 8 57 48 AM

@ekidneyrh I think the design is actually missing a border at the bottom of the header like we have on details pages or the right of the navigation? Easy enough to add here.

2. The article images and buttons are not aligned:
Screenshot 2024-11-27 at 8 45 49 AM

I'm ok if we don't fix here as long as that's a clear requirement in #9704.

If this is the new Dashboard layout, then maybe we should open a followup issue to create a reusable component with the expandable top and layout sections?

**Comment by gastoner** [comment]
image
@deboer-tim bottom border added, Ive keeped the images as tehy are since it should be changed in the https://github.com/podman-desktop/podman-desktop/issues/9704.

**Review by deboer-tim — APPROVED** [review]

**Pull request merged**